docs/api-docs/cart-and-checkout/checkout-sdk.md

```diff
@@ -53,6 +51,17 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
 ## Where can I get the SDK?
 The Checkout JS SDK and associated documentation is available from the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js)
 
+## How can I update the SDK?
+The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.
+
+1. Update the SDK by either running `npm install @bigcommerce/checkout-sdk@<latest-version-numer>` in your terminal, or
+update the version number in your checkout's package.json file, then run npm install.
```

Comment:
* Update the version number in your checkout implementation's package.json file, then run `npm install` in your terminal.